src/utils/index.ts

```
@@ -142,10 +142,10 @@ export const generateUUID = () => {
   const uuid = 'xxxxxxxx-xxxx-4xxx-yxxxx-xxxxxxxxxxxx-xxxxxxxxxxxx-xxxxxx-xxxxxxxx'.replace(/[xy]/g, (c) => {
     let r = Math.random() * 16;
     if (d > 0) {
-      r = ((d + r) % 16) | 0;
+      r = (d + r) % 16 | 0;
```

ESLint

[error] Replace `d·+·r)·%·16` with `(d·+·r)·%·16)`

(prettier/prettier)